package.json

```diff
@@ -97,8 +97,9 @@
     "node-abort-controller": "^3.1.1",
     "opentracing": ">=0.12.1",
     "path-to-regexp": "^0.1.2",
-    "pprof-format": "^2.0.7", 
+    "pprof-format": "^2.0.7",
     "protobufjs": "^7.2.4",
+    "redis-sorted-set": "^2.0.0",
```

Comment:
Currently depends on this feature landing:
https://github.com/axelpale/redis-sorted-set/pull/1

I may need to fork the repo first.